src/what-the-graphlet.h

```diff
@@ -0,0 +1,32 @@
+#ifndef WHAT_THE_GRAPHLET_H
+#define WHAT_THE_GRAPHLET_H
+
+#include "blant.h"
+
+
+#define MAXK 9
```

Comment:
Is there a reason you need this rather than just using MAX_K from blant-fundamentals.h?

Reply:
No

It's just how we did it when coding what-the-graphlet.c, we can use MAX_K

Reply:
Removed it